content/news/2024-06-30-stem-support.md

```diff
@@ -0,0 +1,34 @@
+title: "GSoC 2024: Adding STEM mixing support"
+authors: Antoine Colombier
+tags: gsoc, gsoc-2024, improvements
+slug: STEM-Mixing
+date: 2024-06-30 12:00:00
+
+Disclaimer: *The blog post primarily serves as the documentation for the [Google Summer of Code](https://summerofcode.withgoogle.com/) 2024 project: "Multi channel mixing support (STEMS)".*
+
+## Explore New Creative Horizons with Mixxx's Latest Feature: Stem Mixing
+
+We are excited to introduce a major new feature in Mixxx: Stem Mixing. This innovative tool, inspired by Native Instruments' concept from their Traktor Pro software, allows you to separate and manipulate different elements of a track in real-time. But what exactly are stems?
+
```

Comment:
Before you may mention the origin of the stem system. I think it has been always used in audio productions. The benefit of NI stems is to be accessible during mixing.